docusaurus/docs/reference/metadata.md

```diff
@@ -45,6 +45,7 @@ The `plugin.json` file is required for all plugins. When Grafana starts, it scan
 | `state`              | string                        | No       | Marks a plugin as a pre-release. Possible values are: `alpha`, `beta`.                                                                                                                                                                                                                                                                                                                                  |
 | `streaming`          | boolean                       | No       | For data source plugins, if the plugin supports streaming. Used in Explore to start live streaming.                                                                                                                                                                                                                                                                                                     |
 | `tracing`            | boolean                       | No       | For data source plugins, if the plugin supports tracing. Used for example to link logs (e.g. Loki logs) with tracing plugins.                                                                                                                                                                                                                                                                           |
+| `extensions`         | [object](#extensions)[]       | No       | The list of extensions that the plugin registers under other extension points.                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Use "below" instead of "under" (Google style).

Reply:
Thanks! Fixed in https://github.com/grafana/plugin-tools/pull/1027/commits/d86a27882baf20783c722f78383ef9ee0965a309.